test/test_helper.rb

```diff
@@ -34,12 +34,20 @@
   # Files to track for code coverage
   SimpleCov.track_files("#{src_path}/**/*.rb")
 
-  # use coveralls for on-line code coverage reporting at Travis CI
-  if ENV["TRAVIS"]
-    require "coveralls"
+  # additionally use the LCOV format for on-line code coverage reporting at CI
+  if ENV["CI"] || ENV["COVERAGE_LCOV"]
```

Comment:
why missing mock double configuration here?

Reply:
It's already below...